src/test/java/LaunchingChrome_IT.java

```diff
@@ -0,0 +1,21 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class LaunchingChrome_IT {
+    @Test
+    public void registration() {
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://www.saucedemo.com/");
+        driver.findElement(By.id("user-name")).sendKeys("standard_user");
+        driver.findElement(By.id("password")).sendKeys("secret_sauce");
+        WebElement elementSubmit = driver.findElement(By.id("login-button"));
+        elementSubmit.click();
+        Assert.assertTrue(driver.getPageSource().contains("Product"));
```

Comment:
This is quite strange way of verification. Usually, you have to find something on the page which is present only on that page. Another way is to verify some popup or toast message if possible. Or you can check page title (title in browser tab) if it's unique for each page.

Reply:
In such concrete case it's better to find `Products` element in DOM and then verify if it's displayed.
`driver.findElement(By.xpath("//div[@class='product_label']")).isDisplayed()`

Reply:
Thank you!